src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -843,6 +850,131 @@ public static String composeStreamCallableMethodHeaderSampleCode(
     return bodyStatements;
   }
 
+  private static List<Statement> composeStreamClientSampleCodeBodyStatements(
+      Method method, VariableExpr clientVarExpr, AssignmentExpr requestAssignmentExpr) {
+    List<Expr> bodyExprs = new ArrayList<>();
+
+    // Create responseObserver variable expression.
+    // e.g. ApiStream<EchoResponse> responseObserver
+    TypeNode responseObserverType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(ApiStreamObserver.class)
+                .setGenerics(method.inputType().reference())
+                .build());
+    VariableExpr responseObserverVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("responseObserver").setType(responseObserverType).build());
+    // Create an anonymous class for ApiStreamObserver that contains the methods onNext, onError,
+    // and onCompleted.
+    MethodDefinition onNextMethod =
+        MethodDefinition.builder()
+            .setIsOverride(true)
+            .setScope(ScopeNode.PUBLIC)
+            .setName("onNext")
+            .setArguments(
+                VariableExpr.builder()
+                    .setVariable(
+                        Variable.builder().setName("response").setType(method.outputType()).build())
+                    .setIsDecl(true)
+                    .build())
+            .setBody(
+                Arrays.asList(
+                    CommentStatement.withComment(
+                        LineComment.withComment("Do something when receive a response."))))
```

Comment:
Grammar in comments (might have been wrong in the monolith to begin with): "Do something when a response is received."

Reply:
Done.